sdk/python/feast/permissions/security_manager.py

```diff
@@ -54,7 +50,7 @@ def permissions(self) -> list[Permission]:
         Returns:
             list[Permission]: the list of `Permission` configured in the Feast registry.
         """
-        return self._permissions
+        return self._registry.list_permissions(project=self._project)
```

Comment:
is it ok to list permissions for public access? No need to have elevated permissions?

Reply:
We’re only securing server endpoints, this is an internal API only. Maybe I’m missing your point?